### DISCUSSION TITLE
Should we release 0.67.5?

### DISCUSSION THREAD

**Discussion by ShikaSD** [reactwg/react-native-releases#18] [Patches]
Current Release: [0.67.4](https://github.com/facebook/react-native/releases/tag/v0.67.4)
    
Conversations on this thread are limited: 
* [major release issues](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking)
* [qualified cherry-pick requests](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://github.com/facebook/react-native/wiki/Release-FAQ#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).
    
Please include a link to the specific commit on main to be cherry-picked, for example: https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147
    
In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue. 
If the issue is a [major release issue](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking), please reference the issue here.
    
---
#### List of qualified picks
 
1.
    
#### Local commits to backport to main
1.

**Comment by lachenmayer** [comment]
Hi there, we have come across an issue with `Blob` which is causing 100% reproducible crashes when using `blob.slice` (we are using this to implement chunked/resumable uploads).

The following PR by @awinograd fixes this: https://github.com/facebook/react-native/pull/31392

Apologies that this does not refer to a commit on main, but I do believe that this is breaking a core experience of working in React Native, and has a relatively straightforward fix. We're currently running a patched version of React Native in production, and this works as expected.

The PR & corresponding issue (https://github.com/facebook/react-native/issues/27857) haven't had any activity for several months, so I was hoping to get some eyes on it through this - let me know if there's anything else I can do to get this change released.

Thanks a lot, appreciate it!
